mbtools/utils.py

```diff
@@ -9,3 +9,27 @@ def parse_backup_activities(mbz_dir):
     mbz_path = Path(mbz_dir).resolve(strict=True)
     moodle_backup = models.MoodleBackup(mbz_path)
     return moodle_backup.activities()
+
+
+def find_external_media_references(activity):
+    """Given an activity find external media file references"""
+    media_references = []
+
+    for html_elem in activity.html_elements():
+        media_references += html_elem.find_references_containing(
+            "amazonaws.com"
```

Comment:
This is a good general question. For now I just want to try and consolidate files from those two known external sources as a starting point. It may be that as we continue to develop / QA content we find more sources that we want to include / rewrite. The initial impetus for this was the IM content images as we noticed that in the original content they gave us. The OSX and Moodle plugin media files are two that got added to the scope as content was being built out.